packages/tracing/src/index.ts

```diff
@@ -70,6 +69,7 @@ export const getActiveTransaction = getActiveTransactionT;
  *
  * `extractTraceparentData` can be imported from `@sentry/node`, `@sentry/browser`, or your framework SDK
  */
+// eslint-disable-next-line deprecation/deprecation
 export const extractTraceparentData = extractTraceparentDataT;
```

Comment:
I compared how sentry is initialized on my side with the [documentation](https://docs.sentry.io/platforms/javascript/guides/react/#configure) and BrowserTracing is imported through `@sentry/react`, we were importing it from `@sentry/tracing` directly.

Changing the initialization fixed the problem.

Reply:
Hey, 

so this should still work when being imported from `@sentry/tracing` too - can it be that you had some lingering un-updated versions hanging around? That's usually the reason for these kind of things, and also why we are pushing to having everything imported from a single place (e.g. `@sentry/react`). Probably you had some child dependency of sentry/utils with an older versions where that didn't exist yet!

Reply:
I've looked at the lock file and all sentry related modules where at 7.81.1. I made sure to start from a fresh clone of the projet and cleaned all caches. 

It may be something with yarn, parcel and the way the import made that doesn't work well.

I stopped looking since using the new recommended init works. I just wanted to let you know there might be something here. I wasn't sure if I should fill it as a bug.

Reply:
thanks for bringing this up! We'll keep an eye on this, if this comes up more maybe something _is_ wrong, but "hopefully" it's some weird thing with versions. 👍